dpnp/linalg/linalg_iface.py

```diff
@@ -168,6 +169,30 @@ def eigvals(input):
     return call_origin(numpy.linalg.eigvals, input)
 
 
+def inv(input):
+    """
+        Compute the (multiplicative) inverse of a matrix.
```

Comment:
Please pay your attention to [the docstring template ](https://github.com/IntelPython/dpnp/blob/master/doc/docstring_template.py) when you write docstings.